examples/data-frames.dx

```diff
@@ -396,32 +386,38 @@ def selectColumns {n f'} (f: Fields) (t: n=>{...f & ...f'}) : n=>{&...f} =
   for i.
     {@...f=v, ...} = t.i
     v
--- > selectColumns(students, ["favorite color", "age"])
--- | favorite color | age |
--- | -------------- | --- |
--- | "blue"         | 12  |
--- | "green"        | 17  |
--- | "red"          | 13  |
--- > selectColumns(gradebook, ["final", "name", "midterm"])
--- | final | name    | midterm |
--- | ----- | ------- | ------- |
--- | 87    | "Bob"   | 77      |
--- | 85    | "Alice" | 88      |
--- | 77    | "Eve"   | 84      |
-
-def head = todo
--- > head(students, 1)
--- | name    | age | favorite color |
--- | ------- | --- | -------------- |
--- | "Bob"   | 12  | "blue"         |
--- > head(students, -2)
+ 
+:p selectColumns {favColor:_ ? age:_} students
+:p selectColumns {final:_ ? name:_ ? midterm:_} gradebook
```

Comment:
Doesn't this replace `subTable` from above? I think at some point the benchmark authors might have renamed `subTable` to `selectColumns` and only forgot to update that one example. Let's use this name instead.